Stiletto.KKS/Stiletto.KKS.csproj

```
@@ -0,0 +1,165 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{4F951C42-0F99-43DD-AB03-9724CE2D2924}</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>Stiletto.KKS</RootNamespace>
+    <AssemblyName>KKS_Stiletto</AssemblyName>
+    <TargetFrameworkVersion>v4.6</TargetFrameworkVersion>
+    <FileAlignment>512</FileAlignment>
+    <Deterministic>true</Deterministic>
+    <NuGetPackageImportStamp>
+    </NuGetPackageImportStamp>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
+    <DebugSymbols>true</DebugSymbols>
+    <DebugType>full</DebugType>
```

```suggestion
    <DebugType>pdbonly</DebugType>
```
There are some inconsistent settings between KK and KKS projects, nothing major.

---

I work this and look into the KKS the code is quite different. There was some recent work done one the KK branch that why I pick that one.

---

Also did a fast check on the Stiletto_KKS project the code is to different.  Right now I won't be able to do a good analysis to see what code base is more efficient. I won't promise anything but I'll try to make time.